### DISCUSSION TITLE
print problem

### DISCUSSION THREAD

**Discussion by mariemssi** [jgraph/drawio-desktop#1321] [Q&A]
Hello. I have some days that I have a problem in printing diagram.
The preview of the print doesn't show anything (white paper) and when I launch it I obtain a white paper. could you please, tell me how to solve this?

**Comment by amanda0811** [comment]
It worked normally til last Saturday. I met the same issue and seek for support too. Anybody knows how to fix it please?

**Reply by mariemssi** [reply]
solved
